README.md

```diff
@@ -1,23 +1,110 @@
-```
-import hashlib
-def calculate_md5_checksum(tar_file):
-    hash_md5 = hashlib.md5()
-    with open(tar_file, "rb") as f:
-        for chunk in iter(lambda: f.read(4096), b""):
-            hash_md5.update(chunk)
-    return hash_md5.hexdigest()
-```
-3. After that, create the ```__init__.py``` file for dataset/model. ```__init__.py``` file contains load_data() function. 
-For datasets, this function returns the path to dataset file/folder.
-For model, load_data() fucntion loads and returns the model.
-
-4. Next, create a new release. Add model/dataset name in the fields: tag version and release title.
-5. Then, upload the model/dataset tar.gz file(files in case of multipart) and ```__init__.py``` files and click on publish release.
-6. Lastly, update the list.json file.
+# Gensim data
+This repository contains models and dataset that available through [gensim](https://github.com/RaRe-Technologies/gensim) download api. This repo used as storage.
 
+:red_circle: When you use this API, **all data will be stored in `~/gensim-data` folder**.
+
+## Quickstart
+To load model/dataset you can use 2 types of API:
+- **Python API**
+
+  Example: load pre-trained vectors
+  ```python
+  import gensim.downloader as api
+
+  info = api.info()  # show info about available models/datasets
+  model = api.load("glove-twitter-25")  # download model and load it to memory
+  model.most_similar("cat")
+  
+  """
+  output:
+
+  [(u'dog', 0.9590819478034973),
+   (u'monkey', 0.9203578233718872),
+   (u'bear', 0.9143137335777283),
+   (u'pet', 0.9108031392097473),
+   (u'girl', 0.8880630135536194),
+   (u'horse', 0.8872727155685425),
+   (u'kitty', 0.8870542049407959),
+   (u'puppy', 0.886769711971283),
+   (u'hot', 0.8865255117416382),
+   (u'lady', 0.8845518827438354)]
+ 
+  """
+  ```
+  
+  Example: load dataset and train Word2Vec
+  ```python
+  from gensim.models.word2vec import Word2Vec
+  import gensim.downloader as api
+
+  corpus = api.load('text8')  # download dataset and load iterable to memory
+  model = Word2Vec(corpus)  # train model
+  model.most_similar("car")
+
+  """
+  output:
+
+  [(u'driver', 0.8273754119873047),
+   (u'motorcycle', 0.769528865814209),
+   (u'cars', 0.7356342077255249),
+   (u'truck', 0.7331641912460327),
+   (u'taxi', 0.718338131904602),
+   (u'vehicle', 0.7177008390426636),
+   (u'racing', 0.6697118878364563),
+   (u'automobile', 0.6657308340072632),
+   (u'passenger', 0.6377975344657898),
+   (u'glider', 0.6374964714050293)]
+ 
+  """
+  ```
+  
+  Example: how to **only** download data (without loading to memory)
+  ```
+  import gensim.downloader as api
+  
+  print(api.load("20-newsgroups", return_path=True))  # output: /home/user/gensim-data/20-newsgroups/20-newsgroups.gz
+  print(api.load("glove-twitter-25", return_path=True))  # output: /home/user/gensim-data/glove-twitter-25/glove-twitter-25.gz
```

Comment:
Missing file extension -- if this is a text file, should be `.txt.gz` (here and elsewhere).

By convention, files without an extension are binary files (which is not the case here).

Reply:
It's impossible right now, we used common substitution like `<data-name>.gz` to retrieve a link.

Reply:
OK.